docs/index.md

```diff
@@ -1,80 +1,39 @@
-
-#### Main Categories
-
-* [Components](components/index.md)
-	* [Screen](components/screen.md)
-	* [User Interface](components/user-interface/index.md)
-	* [Layout](components/layout/index.md)
-	* [Media](components/media/index.md)
-	* [Drawing and Animation](components/drawing-and-animation/index.md)
-	* [Sensors](components/sensors/index.md)
-	* [Social](components/social/index.md)
-	* [Storage](components/storage/index.md)
-	* [Device](components/device/index.md)
-	* [Dynamic Components](components/dynamic/index.md)
-	* [Connectivity](components/connectivity/index.md)
-	* [Google](components/google/index.md)
-	* [Monetization](components/monetization/index.md)
-	* [LEGO Mindstorms](components/lego-mindstorms/index.md)
-	* [Experimental](components/experimental/index.md)
-* [Blocks](blocks/index.md)
-	* [Control](blocks/control.md)
-	* [Logic](blocks/logic.md)
-	* [Math](blocks/math.md)
-	* [Text](blocks/text.md)
-	* [Lists](blocks/lists.md)
-	* [Colors](blocks/colors.md)
-	* [Variables](blocks/variables.md)
-	* [Procedures](blocks/procedures.md)
-	* [Any Component](blocks/any-component.md)
-
-#### Testing
-
-* [Live Development](live-development/index.md)
-	* [Via WiFi](live-development/wifi.md)
-	* [Via USB](live-development/usb.md)
-
-#### Extension
-
-* [Overview](extensions/index.md)
-* [Create Extensions](extensions/create-extensions.md)
-
-#### Other
-
-* [Resources](other/resources/index.md)
-	* [Create Extensions](other/resources/concept-cards.md)
-* [Release Notes](other/release-notes/index.md)
-* [Backpack](other/backpack.md)
-* [Terms of Service](other/terms-of-service.md)
-* [Donate](https://paypal.me/Makeroid)
-
-## Makeroid
-
-Makeroid allows you to create Android apps easily with a blocks-type editor. No coding skills required. With the Material Design UI, your apps will stand out. We always listen to our users' opinions from our community \(your suggestions, bug reports etc.\).
+Whether you need some help using the components, or just looking around, this section will provide you the guides and reference you need to develop an app.
+
+For further and personalized help, consider checking out [our community](https://community.makeroid.io).
+
+## Get Started
+Start by searching what you need using the search bar at the top. You can also consider checking out [Components](https://docs.makeroid.io/components/) or [Blocks](https://docs.makeroid.io/blocks/).
+
+## What is Makeroid?
+
+Makeroid allows you to create Android apps easily with a blocks-type editor. No coding skills required. With the Material Design UI, your apps will stand out.
 
 ## Useful Links
 
-### [Makeroid Builder](http://builder.makeroid.io)
+### [Builder](http://builder.makeroid.io)
 
 This is the page you're going to use to build apps. This is main app that allows you to create anything you want with a Material Design UI.
 
-### [Makeroid Community](https://community.makeroid.io)
+### [Community](https://community.makeroid.io)
```

Comment:
Move Community to the top of Useful Links: Community | Builder | IDE | Store | Status

Reply:
https://github.com/Makeroid/Documentation/commit/189f44ec3fb3636538d4408e4241dd0c2a06402f